orangecontrib/timeseries/timeseries.py

```diff
@@ -1,13 +1,49 @@
-
+import itertools
 import numpy as np
 
-from Orange.data import Table, Domain, TimeVariable
+from Orange.data import Table, Domain, TimeVariable, ContinuousVariable
 
 import Orange.data
 from os.path import join, dirname
 Orange.data.table.dataset_dirs.insert(0, join(dirname(__file__), 'datasets'))
 
 
+class TimeDelta:
```

Comment:
This probably shouldn't be in this PR! 😅

Reply:
Gave a git alias the wrong commit hash, always git log before git push kids